db/migrate/20230503143849_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
```

Comment:
- [ ] Same here about references, it should be referred to post and author like `t.references :author, foreign_key: { to_table: 'users' }` and `t.references :post, foreign_key: true`. this will create `author_id`, `post_id`, and a corresponding index. Kindly add them 🙃